clang/test/CAS/mccas-replay-test.cpp

```diff
@@ -0,0 +1,43 @@
+// RUN: rm -rf %t && mkdir -p %t
+// RUN: export LLVM_CACHE_CAS_PATH=%t/cas && %clang-cache \
+// RUN:   %clang -target arm64-apple-macosx12.0.0 -c -Xclang -fcas-backend -Rcompile-job-cache %s -o %t/tmp.o -g 2>&1 | FileCheck %s -check-prefix=CACHE-MISS
+// CACHE-MISS: remark: compile job cache miss
+
+// RUN: llvm-objdump -h %t/tmp.o | FileCheck %s -check-prefix=CACHE-MISS-FILE
```

Comment:
You don't need to have separate `CACHE-MISS-FILE` and `CACHE-HIT-FILE`. They are the same. You can reuse the same check.